### DISCUSSION TITLE
Please add prop to disable changing value of input while hovering days in datepicker

### DISCUSSION THREAD

**Discussion by dreamerrrrrrrrr** [formkit/formkit#1242] [Ideas]
https://github.com/formkit/formkit/assets/162573544/9c73bbf2-f89f-4ad7-901a-11ec5181f934

**Comment by fenilli** [comment]
It is only a visual help to see how the date will look after setting, if you click outside of the box so not setting any value, the value won't actually change.

**Reply by dreamerrrrrrrrr** [reply]
I know, but it can be confusing for some users
